src/steamship/agents/llms/openai.py

```diff
@@ -80,7 +82,14 @@ def chat(self, messages: List[Block], tools: Optional[List[Tool]], **kwargs) ->
         - `max_tokens` (controls the size of LLM responses)
         """
 
-        temp_file = File.create(client=self.client, blocks=messages)
+        temp_file = File.create(
+            client=self.client,
+            blocks=messages,
+            tags=[
+                # Mark that this is a CHAT COMPLETION file
+                Tag(kind=TagKind.CHAT, name=ChatTag.COMPLETION)
+            ],
+        )
 
```

Comment:
Why mark this 'temp_file' as a chat completion?  If anything is a completion, shouldn't it be the generated block? 

Seems like maybe you want a new tag kind for this file.

Reply:
Just switched to:

```
  Tag(kind=TagKind.GENERATION, name=GenerationTag.PROMPT_COMPLETION)
```